src/main/java/net/consensys/orion/http/handler/privacy/ModifyPrivacyGroupRequest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.orion.http.handler.privacy;
+
+import java.io.Serializable;
+import java.util.Optional;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+public class ModifyPrivacyGroupRequest implements Serializable {
```

Comment:
With respect to this pull request it only appends and adds, however I have future pull requests (already implemented but not yet ready to merge) incoming where it will rename, and do various other things which modify the group.

Reply:
It might make sense to segregate this. I think we should bear this in mind for the future PR where we can have some more context.